sigmet/sigmet.py

```diff
@@ -1,11 +1,43 @@
-
-
 class Sigmet:
 
     def __init__(start_point, end_point, data):
         this.start_point = start_point
         this.end_point = end_point
         this.data = data
 
-    def fit(cutoff_date, p, q, d):
-        pass
+
+
+    def fit(self, cutoff_date, p, q, d, standardize=False):
+    """
+    Fits the pandas.Series object
+    
+    Parameters
+    __________
+    
+    cutoff_date : 
+    
+    p :
+    
+    q :
+    
+    d :
+    
+    standardize : boolean object, default=False
+        If False, then no change to the fitted Series.
+        If True, then the fitted Series will be standardized before being passed into .fit() .
+        
+    Returns
+    _______
+    
+    self : object
+        Returns an instance of self.
+    """
+
+    srs = self.copy(deep=True)
+    
+    if standardize == True:
+        temp = standardize(srs)
```

Comment:
should be srs, that was srs will contain the standardized or non-standardized data, depending on user preference